Training/1.0 Introduction.md

```diff
@@ -12,37 +12,69 @@ First we we learn a little bit about coding in Python, although many of you will
 List all the Data Types of Python with examples.  
 
 ```
-
+str:'hello'
+int:1
+float:2.3
+bool:True
+complex:1j
+list:[1,2,3]
+tuple:('a','s,'b')
+range:range(5)
+dict:{'name':'Murat','age':23}
+set:{"apple", "banana", "cherry"}
+frozenset:frozenset({"apple", "banana", "cherry"})
+bytes:b"Hello"
+bytearray:bytearray(5)
+memoryview:memoryview(bytes(5))
+NoneType:None
```

Comment:
Good job, you'll mainly need the first 9 for most general purposes including HDA.